test/k8s-integration/main.go

```diff
@@ -436,10 +439,20 @@ func handle() error {
 
 	switch testParams.deploymentStrategy {
 	case "gke":
-		testParams.cloudProviderArgs, err = getGKEKubeTestArgs(*gceZone, *gceRegion, testParams.imageType)
-		if err != nil {
+		// Build params for both kubetest and kubetest2 for debugging.
+		testParams.cloudProviderArgs, err = getGKEKubeTestArgs(*gceZone, *gceRegion, testParams.imageType, false /* useKubeTest2 */)
```

Comment:
If we don't even log an error when, eg kubetest2=false and kubetest2 arg building returns an error, why are we building both sets of args?

It may simplify the logic considerably to just plumb through one set of args.

Reply:
Agreed - changed to only generate one set of args.